### PR TITLE
add clarity to use of dask on HPCs

### PR DESCRIPTION
**Pull request by amsnyder**
Trying to address the lack of clarity mentioned in #510

### CODE REVIEW

File: environment_set_up/OpenOnDemand.md
Comment (pnorton-usgs) [inline environment_set_up/OpenOnDemand.md:1]: Suggest changing:
```
(1) You can use the form to request the number of cores you need and then run a [Dask Local Cluster](./Start_Dask_Cluster_Denali.ipynb) on those cores. Or you can request...
```
to
```
(1) You can use the form to request the number of cores you need and then run a [Dask Local Cluster](./Start_Dask_Cluster_Denali.ipynb) on those cores, or (2) you can request...
```

Suggest changing:
```
Submit
```
to
```
Click Submit
```

**Review by pnorton-usgs — APPROVED** [review]
Just a couple minor changes, otherwise it looks good.

File: environment_set_up/OpenOnDemand.md [hunk 1]
```diff
@@ -3,8 +3,10 @@
...
+* Fill in the form to customize the allocation in which the Jupyter Server will execute.
+  * You may want to consider adding the git and/or aws modules if you plan to use them during your session. You will just need to type `module load git` and/or `module load aws` in the `Module loads` section.
+  * If you expect to run code in parallel on multiple compute nodes, you have two options. (1) You can use the form to request the number of cores you need and then run a [Dask Local Cluster](./Start_Dask_Cluster_Denali.ipynb) on those cores. Or you can request the standard 2 cores, and then use a [Dask SLURMCluster](./Start_Dask_Cluster_Tallgrass.ipynb) to submit new jobs to the SLURM scheduler, giving you access to additional compute nodes.
```
Comment (amsnyder) [inline environment_set_up/OpenOnDemand.md:9]:
```suggestion
  * If you expect to run code in parallel on multiple compute nodes, you have two options. (1) You can use the form to request the number of cores you need and then run a [Dask Local Cluster](./Start_Dask_Cluster_Denali.ipynb) on those cores, or (2) you can request the standard 2 cores, and then use a [Dask SLURMCluster](./Start_Dask_Cluster_Tallgrass.ipynb) to submit new jobs to the SLURM scheduler, giving you access to additional compute nodes.
```

File: environment_set_up/OpenOnDemand.md [hunk 2]
```diff
@@ -3,8 +3,10 @@
...
 * Submit
```
Comment (amsnyder) [inline environment_set_up/OpenOnDemand.md:10]:
```suggestion
* Click Submit
```

File: environment_set_up/OpenOnDemand.md
Comment (amsnyder) [reply environment_set_up/OpenOnDemand.md:1]: Fixed! thanks!

**Pull request merged**